extras/courses.md

```diff
@@ -90,7 +90,7 @@ Courses | Duration | Effort
 
 Courses | Duration | Effort
 :-- | :--: | :--:
-[How to Use Git and GitHub](https://www.udacity.com/course/how-to-use-git-and-github--ud775) | 3 weeks | 2-3 hours/week
+[How to Use Git and GitHub](https://www.udacity.com/blog/2015/06/a-beginners-git-github-tutorial.html) | 3 weeks | 2-3 hours/week
```

Comment:
The new link is just a blog article with an estimated 12 min read time, hence makes the duration and effort wrong.

Reply:
It is true that I mentioned that it is not the original course because the link to the original course is invalid, but I put this link to the same site, but if you notice at the end of the article there are additional links for you to learn in more depth.